### PR TITLE
Release notes for SC 2019/09

### PR DESCRIPTION
**Pull request by pcm32**
This PR starts the release notes. All numerical values have been updated, New features section needs to be completed still.

New experiments where generated considering:
```
echo '"28-06-2019"' | jq 'strptime("%d-%m-%Y")|mktime'
```

### CODE REVIEW

File: release-notes/sc/_posts/2019-09-16-7.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- [scRNA-seq to investigate endoderm differentiation from induced human pluripotent stem cells](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7008)
+- [Origin and differentiation trajectories of fibroblastic reticular cells in the splenic white pulp - single cell RNAseq dataset 1](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7094)
+- [Single-cell RNA-seq of CD4+ Treg and Tmem cells from different tissues in mouse (Chromium 10x data)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7311)
+- [Single cell RNA-seq of adult human neural retina](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7316)
+
+
+
+#### New features
+
+Release 7 is essentially a data update release, so no new software features were incorporated.
```
Comment (suhaibMo) [inline release-notes/sc/_posts/2019-09-16-7.md]: Are we not releasing software along with this release. @alfonsomunozpomer ?? Perhaps we can  add software updates into same PR ?

Comment (alfonsomunozpomer) [inline release-notes/sc/_posts/2019-09-16-7.md]: I would ommit this block. If there’s nothing to say, I think it’s better not to say anything. ;)

Comment (alfonsomunozpomer) [reply release-notes/sc/_posts/2019-09-16-7.md]: There’s a new software version, but no new features in terms of the UI. Some fixes and mostly performance improvements. Handling of `load_date` is not something I would advertise, either.

Comment (pcm32) [reply release-notes/sc/_posts/2019-09-16-7.md]: Yes, I was between that and what I wrote, agreed, will remove it.

**Comment by pcm32** [comment]
Thanks for reviewing @alfonsomunozpomer @suhaibMo

**Pull request merged**